qiskit_optimization/algorithms/__init__.py

```diff
@@ -53,6 +53,8 @@
    MinimumEigenOptimizationResult
    MinimumEigenOptimizer
    OptimizationResultStatus
+   QuantumRandomAccessOptimizationResult
+   QuantumRandomAccessOptimizer
```

Comment:
I guess I was more thinking a module level link, similar to what we have at the higher level for algorithms, applications, problems submodules. That would then take you to init file where you have all those classes documented and that would be where you would import them from. I think Sphinx will be unhappy if we document things in two places (I have seen that before as an error). Since there are a bunch of classes you would need to have .qrao as well in the import without some convenience import here - but its short right and quick to type so I think its ok and it keeps all those related classes together both in terms of importing them and in regards of where they are documented.